.travis.yml

```diff
@@ -12,7 +12,7 @@ services:
 - docker
 language: go
 go:
-- 1.12.9
+- 1.14.7
```

Comment:
@mittachaitu we have to so some more refactring similar to zfs localpv, follow the Makefile changes from this PR https://github.com/openebs/zfs-localpv/pull/148/files